mods/ts/rules/nod-vehicles.yaml

```diff
@@ -256,7 +258,7 @@ STNK:
 		Cost: 1100
 	Tooltip:
 		Name: Stealth Tank
-		Description: Long-range missile tank that can cloak.\nHas weak armor. Can be spotted by infantry.\n  Strong vs Vehicles, Tanks, Aircraft\n  Weak vs Infantry.
+		Description: Long-range missile tank that can cloak.\nHas weak armor. Can be spotted by infantry.\n  Strong vs Vehicles, Tanks, Aircraft\n  Weak vs Infantry
```

Comment:
I don't recall it being very long-ranged :?
Also "Can be spotted by infantry at close range".
And "Strong vs Vehicles, Tanks, Aircraft" has one ground-based group of units too much :P